client/app/host/page.tsx

```diff
@@ -1,3 +1,368 @@
+    const [teamList, setTeamList] = useState([
+        { name: 'Team1', password: 'password', points: 300, status: true },
+        { name: 'Team2', password: 'password', points: 126, status: true },
+        { name: 'Team3', password: 'password', points: 0, status: false },
+        { name: 'Team4', password: 'password', points: 299, status: true },
+        { name: 'Team5', password: 'password', points: 0, status: true },
+        { name: 'Team6', password: 'password', points: 5, status: false },
+        { name: 'Team7', password: 'password', points: 125, status: true },
+    ]);
+    const [errorMessage, setErrorMessage] = useState('');
+
+    useEffect(() => {
+        setIsServerOn(true);
+    }, []);
+
+    const handleNewQuestionClick = () => {
+        if (!newQuestionText || !newQuestionPoints || !selectedLanguage) {
+            setErrorMessage('Please Fill In All Fields');
+            return;
+        }
+
+        setQuestions((prevQuestions) => [
+            ...prevQuestions,
+            { question: newQuestionText, language: selectedLanguage, points: newQuestionPoints },
+        ]);
+        setNewQuestionText('');
+        setNewQuestionPoints('');
+        setSelectedLanguage('');
+        setErrorMessage('');
+    };
+
+    const handleAddTeam = (data: { name: string; password: string }): boolean => {
+        const isUnique = !teamList.some(
+            (team) => team.name.toLowerCase() === data.name.toLowerCase()
+        );
+
+        if (isUnique) {
+            setTeamList((prevTeams) => [
+                ...prevTeams,
+                { name: data.name, password: data.password, points: 0, status: false },
+            ]);
+        }
+        return isUnique;
+    };
+
+    const disconnectAllTeams = () => {
+        const updatedTeams = teamList.map((team) => ({
+            ...team,
+            status: false,
+        }));
+        setTeamList(updatedTeams);
+    };
+
+    const handleDisconnectTeam = (teamName: string) => {
+        setTeamList((prev) =>
+            prev.map((team) => (team.name === teamName ? { ...team, status: false } : team))
+        );
+    };
+
+    const handleRemoveTeam = (teamName: string) => {
+        setTeamList((prev) => prev.filter((team) => team.name !== teamName));
+    };
+
+    const handleRemoveQuestion = (q: string) => {
+        setQuestions((prev) => prev.filter((question) => question.question !== q));
+    };
+
+    const handleToggleServer = () => {
+        setIsServerOn((prev) => (prev === null ? true : !prev));
+        disconnectAllTeams();
+    };
+
+    return (
+        <ResizablePanelGroup direction="horizontal" className="flex max-h-screen flex-grow">
+            <ResizablePanel className="flex flex-col justify-center" defaultSize={30} maxSize={50}>
+                <div className="flex h-fit w-full justify-evenly pb-2 pt-4">
+                    <AddTeamDialog onAddTeam={handleAddTeam} />
+                    <p className="px-14 text-2xl uppercase">Teams</p>
```

Comment:
Remove some of the padding on 'Teams' and use justify-between so that the buttons are on the outer edges.